test/pantry-test.js

```diff
@@ -83,4 +98,8 @@ describe('Pantry', () => {
     
     expect(pantry.checkPantry(extraIngredientRecipe)).to.eql(missingIngredientsWithPrice);
   });
+
+  it('If no recipe ingredients are given then the method checkPantry should give an Error', () => {
+    expect(() => { pantry.checkPantry() }).to.throw(Error);
+  });
```

Comment:
Sad paths added for the method and then class itself.

Reply:
Maybe we can have it return null if no ingredients are given? or if try to pass in other datatypes it could also return null